### PR TITLE
817 drill down table

### PR DESCRIPTION
**Pull request by p-netm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,6 +64,8 @@
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.3",
     "react-select": "^3.0.4",
+    "react-table": "^7.0.4",
+    "react-table-v6": "^6.8.6",
```
Comment (moshthepitt) [inline package.json:68]: Do we still need react-table-v6?

File: src/containers/pages/FocusInvestigation/jurisdiction/index.tsx [hunk 2]
```diff
@@ -250,13 +249,9 @@ export const FIJurisdiction = (props: FIJurisdictionProps & RouteComponentProps<
...
         /** Handle all Columns i.e columns unique for reactive and routine (once handled above) plus columns common on both (name)  */
-        const allColumns: Column[] = [
-          ...jsxColumns('name'),
-          ...statusColumn,
-          ...columnsBasedOnReason,
-        ];
+        const allColumns = [...jsxColumns('name'), ...statusColumn, ...columnsBasedOnReason];
         /** Contains columns and data that will build the table */
-        const tableProps = {
+        const tableProps: any = {
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/jurisdiction/index.tsx:254]: Does this necessarily have to be of type `any`?

**Comment by p-netm** [comment]
@moshthepitt RFC
Screenshot from 2020-06-17 14-34-49
Screenshot from 2020-06-17 14-35-12
Screenshot from 2020-06-17 14-35-19

**Comment by moshthepitt** [comment]
@p-netm can we change the table widths on manage plans such that "Intervention Type" is on one line?

**Comment by p-netm** [comment]
> @p-netm can we change the table widths on manage plans such that "Intervention Type" is on one line?

cool.

**Comment by moshthepitt** [comment]
@p-netm also looks like the "Last Modified" column need not be that wide

**Comment by p-netm** [comment]
> @p-netm also looks like the "Last Modified" column need not be that wide

sawa

**Comment by p-netm** [comment]
Screenshot from 2020-06-19 19-40-51

I

**Comment by Naima-Bashir** [comment]
This has passed QA

**Pull request merged**